x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/type/AbstractEsFieldTypeTests.java

```diff
@@ -32,6 +41,25 @@ public static EsField randomAnyEsField(int maxDepth) {
 
     protected abstract T mutate(T instance);
 
+    @Override
+    protected EsField copyInstance(EsField instance, TransportVersion version) throws IOException {
+        NamedWriteableRegistry namedWriteableRegistry = getNamedWriteableRegistry();
+        try (
+            BytesStreamOutput output = new BytesStreamOutput();
+            var pso = new PlanStreamOutput(output, new PlanNameRegistry(), EsqlTestUtils.TEST_CFG)
```

Comment:
I had to move these tests to `esql` module to use `PlanStreamOutput`/`Input`.
This is a bit unfortunate, but probably we are close to merging the two modules anyway

Reply:
I don't mind moving them.